### PR TITLE
Task/header 2fa

### PR DESCRIPTION
**Pull request by catherine-jones**
- create markup for new header
- styles new header
- includes new header in devise and editor views

### CODE REVIEW

File: client/src/editor/images/dashboardlogo.svg [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (tobyhede) [inline client/src/editor/images/dashboardlogo.svg]: this seems like a LOT of SVG. What does it actually render?

File: app/views/devise/two_factor_authentication/_govau-header.erb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<div class="govau-header-row" role="banner">
+  <div class="wrapper">
+    <header class="govau-masthead">
+      <div class="banner">
+        <div class="brand">
+          <a href="/">
```
Comment (elisechant) [inline app/views/devise/two_factor_authentication/_govau-header.erb]: href should be `/editor`

File: app/views/devise/two_factor_authentication/_govau-header.erb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+            <img src="./client/src/editor/images/gov-performance-editor-logo.svg" alt="Logo of GOV.AU Performance Editor" height="30px" />
```
Comment (elisechant) [inline app/views/devise/two_factor_authentication/_govau-header.erb]: src value here is not correct, just use the Rails helpers like normal

File: app/views/partials/_govau-header.html.erb [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<div class="govau-header-row" role="banner">
+  <div class="wrapper">
+    <header class="govau-masthead">
+      <div class="banner col-sm-9">
+        <div class="brand">
+          <a href="/">
+            <img src="../../../client/src/editor/images/dashboardlogo.svg" alt="Logo of GOV.AU Performance Editor" height="30px" />
```
Comment (elisechant) [inline app/views/partials/_govau-header.html.erb]: this src is wrong here as well

File: app/views/partials/_govau-header.html.erb [hunk 5]
```diff
@@ -0,0 +1,18 @@
+<div class="govau-header-row" role="banner">
+  <div class="wrapper">
+    <header class="govau-masthead">
+      <div class="banner col-sm-9">
```
Comment (elisechant) [inline app/views/partials/_govau-header.html.erb]: can't use columns without a row

File: client/src/editor/images/dashboardlogo.svg (hunk 1 above)
Comment (elisechant) [inline client/src/editor/images/dashboardlogo.svg]: It's an external asset so I suppose it is reasonable.

File: client/src/editor/images/header-bg.svg [hunk 6]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (elisechant) [inline client/src/editor/images/header-bg.svg]: Is this the gradient? not crazy about this or about client/src/editor/images/header-bg.jpg. What reasonable alternatives are there?

File: client/src/login/styles/vendor/_govau-header-overrides.scss [hunk 7]
```diff
@@ -0,0 +1,22 @@
+.govau-masthead .banner {
+  display:block;
+}
+
+.sign-out {
+  float: right;
+
+  .user-email {
+    &::before {
+    background-image: url("user.svg");
```
Comment (elisechant) [inline client/src/login/styles/vendor/_govau-header-overrides.scss]: is this an icon?

File: client/src/login/styles/vendor/_govau-header.scss [hunk 8]
```diff
@@ -0,0 +1,267 @@
...
+
+
+//masthead
+//=======================
+//Grouping of <header> elements, some of which may fall outside of .banner
+
+.govau-masthead {
+  @include clearfix;
+
+  @include media($tablet) {
```
Comment (elisechant) [inline client/src/login/styles/vendor/_govau-header.scss]: is `$tablet` new?

File: app/views/partials/_govau-header.html.erb (hunk 5 above)
Comment (catherine-jones) [inline app/views/partials/_govau-header.html.erb]: thanks for the reminder

File: client/src/editor/images/header-bg.svg (hunk 6 above)
Comment (catherine-jones) [inline client/src/editor/images/header-bg.svg]: this is something that had come from UI kit apparently there is an issue with using gradients that is notes in the code

File: db/schema.rb [hunk 9]
```diff
@@ -67,13 +67,13 @@
...
-    t.text     "name",            null: false
-    t.text     "label",           null: false
-    t.text     "units",           null: false
+    t.text     "name",                              null: false
+    t.text     "label",                             null: false
+    t.text     "units",                             null: false
     t.text     "notes"
-    t.datetime "created_at",      null: false
-    t.datetime "updated_at",      null: false
-    t.string   "period"
```
Comment (tobyhede) [inline db/schema.rb]: Don't think this should be in here. Looks like some changes from another branch.

Comment (catherine-jones) [inline db/schema.rb]: is it not in master?

**Comment by elisechant** [comment]
@catherine-jones if you pop images in app/assets/images and then use the Rails image helpers you should be sweet for logo. For icons like avatar icon, come and talk to me we have an icon generator thingo now.

**Comment by elisechant** [comment]
this branch needs to rebase with master. it keeps failing builds because environment has changed since this branch was branched.

**Comment by elisechant** [comment]
closing because https://github.com/AusDTO/dto-dashboard/pull/452

**Pull request closed without merging**